src/clr-addons/history/history-pinned.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2018-2020 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, Input, OnInit, OnDestroy } from '@angular/core';
+import { ClrHistoryModel } from './history-model.interface';
+import { ClrHistoryService } from './history.service';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'clr-history-pinned',
+  templateUrl: './history-pinned.html',
+})
+export class ClrHistoryPinned implements OnInit, OnDestroy {
+  @Input('username') username: string;
```

Comment:
we follow clarity styleguides, so input variables should prefix 'clr' (clrUsername in this case)
https://github.com/vmware/clarity/blob/master/docs/CODING_GUIDELINES_ANGULAR.md